WordPress/Classes/ViewRelated/Stats/Subscribers/StatsSubscribersLineChart.swift

```diff
@@ -0,0 +1,36 @@
+import Foundation
+import DGCharts
+
+private struct SubscriberLineChartData: LineChartDataConvertible {
+    let accessibilityDescription: String
+    let lineChartData: LineChartData
+}
+
+class StatsSubscribersLineChart {
+
+    let lineChartData: LineChartDataConvertible
+    let lineChartStyling: LineChartStyling
+
+    init(counts: [Int]) {
+        let chartEntries = counts.enumerated().map { index, count in
+            ChartDataEntry(x: Double(index), y: Double(count))
+        }
+        let dataSet = LineChartDataSet(entries: chartEntries)
+        let chartData = LineChartData(dataSets: [dataSet])
+        lineChartData = SubscriberLineChartData(accessibilityDescription: "Subscriber Charts", lineChartData: chartData)
+        lineChartStyling = SubscribersLineChartStyling()
+    }
+}
+
+// MARK: - StatsSubscribersLineChartStyling
+
+private struct SubscribersLineChartStyling: LineChartStyling {
+    let primaryLineColor: UIColor                   = UIColor(light: .muriel(name: .blue, .shade50), dark: .muriel(name: .blue, .shade50))
```

Comment:
I couldn't find equivalents for these (which are from `ColorPalette.xcassets`) in the design system `Colors.xcassets`. For example, this `primaryLineColor` or `blue50` is:
- RGB 196, 117, 6 (light) and 227, 156, 57 (dark) using old system
- RGB 227, 156, 57 (light and dark) using the new system 

Please let me know if I'm missing something, otherwise I think keeping this as-is is fine.

Reply:
pinging you @staskus in case you have thoughts

Reply:
Yeah, I don't know about that. Need to speak with a designer to see if we want to use colors from the Design System in both cases.

Reply:
I'm going to keep the old colors as this was done on Android "I didn’t change colors to align with the current design",  see p1713970560616449/1713880076.221049-slack-C06BR07TJHK